library/gen_x12_837.inc.php

```diff
@@ -89,35 +89,9 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim = false)
     ++$edicount;
 
     if ($claim->federalIdType() == "SY") { // check entity type for NM*102 1 == person, 2 == non-person entity
-        $tempName = $claim->billingFacilityName();
```

Comment:
ok, thanks will keep cleaning up the area probably only to be trumped by Sena's gang but will hopefully be good learning for me